### ISSUE TITLE
optimize refresh preview 

### ISSUE THREAD

**Issue by wu4339** [hardcore-sushi/DroidFS#264]
Each open there is a refresh preview of the process, whether it can be optimized, so that the experience will be better

**Comment by hardcore-sushi** [comment]
Yes it's already planned: [TODO.md](https://github.com/hardcore-sushi/DroidFS/blob/master/TODO.md#health)

**Comment by learnmlcolab** [comment]
@hardcore-sushi I saw a comment from you [here](https://github.com/hardcore-sushi/DroidFS/issues/219#issuecomment-1628650021).

> Storing them encrypted in a hidden folder in the volume would modify the volume without the user's decision, which is not what we want. 

Would you consider this option? I'm thinking the app could ask the user for permission to store thumbnails in a ` .thumbnails` folder at the root of the volume or something like that.

**Comment by hardcore-sushi** [comment]
v2.3.0 introduces a far better cache for both thumbnails and full image displays. Let me know if that's enough to close this issue or if improvements are still necessary.
